packages/patient-portal/src/App.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import { BrowserRouter as Router, Routes, Route, Navigate } from 'react-router';
-import { LoginView, RegistrationView, DashboardView } from './views';
-import { PublicRoute } from './routes/PublicRoute';
-import { PrivateRoute } from './routes/PrivateRoute';
+import { LoginView, RegistrationView, DashboardView, RequestLoginTokenView } from './views';
+import { PublicRoute } from '@routes/PublicRoute';
+import { PrivateRoute } from '@routes/PrivateRoute';
 
-export const RoutingApp = React.memo(() => {
+export const App = () => {
   return (
     <Router>
       <Routes>
         <Route element={<PublicRoute />}>
-          <Route path="/login" element={<LoginView />} />
+          <Route path="/login" element={<RequestLoginTokenView />} />
+          <Route path="/login-submit" element={<LoginView />} />
```

Comment:
I'm open to suggestions about these route urls. I just wanted the first screen to be /login so that it would look good for links to login urls that might be sent to users